src/Features/Core/Portable/Completion/CompletionService.cs

```diff
@@ -169,5 +170,45 @@ public virtual TextSpan GetDefaultCompletionListSpan(SourceText text, int caretP
 
             return bestItems.ToImmutableAndFree();
         }
+
+        internal static async Task<CompletionDescription> GetDescriptionAsync(
+            CompletionService service, CompletionItem item, CancellationToken cancellationToken = default)
+        {
+            var document = item?.Document;
+            if (document == null || service == null)
+            {
+                return CompletionDescription.Empty;
+            }
+
+            return await service.GetDescriptionAsync(item.Document, item, cancellationToken).ConfigureAwait(false);
+        }
+
+        internal static async Task<CompletionList> GetCompletionsAsync(
```

Comment:
Having a static method on `CompletionService` that requires an instance of a `CompletionService` is just wrong.